File: server/SantralOpsAPI/Entities/Ticket.cs
using SantralOpsAPI.Enums;

namespace SantralOpsAPI.Entities;

public class Ticket : BaseEntity
{
  public string Konu { get; set; }
  public string Aciklama { get; set; }
  public TicketDurum Durum { get; set; } = TicketDurum.Acik;
  public TicketOncelik Oncelik { get; set; } = TicketOncelik.Normal;
  public int TalebiYapanKisiId { get; set; }
  public virtual Kisi TalebiYapanKisi { get; set; }
  public int OlusturanPersonelId { get; set; }
  public virtual Personel OlusturanPersonel { get; set; }
  public int? AtananPersonelId { get; set; }
  public virtual Personel? AtananPersonel { get; set; }
}

File: client/src/types/ticket.ts
export type Ticket = {
  id: number;
  konu: string;
  aciklama: string;
  oncelik: string;
  durum: string;
  talebiYapanKisiId: string;
  talebiYapanKisi: string;
  olusturanPersonelId: string | null;
  olusturanPersonel: string | null;
  atananPersonelId: string;
  atananPersonel: string;
  createdDate: string;
};
